include/deal.II/lac/petsc_compatibility.h

```diff
@@ -53,7 +53,7 @@ namespace PETScWrappers
 #if DEAL_II_PETSC_VERSION_LT(3, 7, 0)
     const PetscErrorCode ierr = PetscOptionsSetValue (name.c_str (), value.c_str ());
 #else
-    const PetscErrorCode ierr = PetscOptionsSetValue (NULL, name.c_str (), value.c_str ());
+    const PetscErrorCode ierr = PetscOptionsSetValue ((nullptr), name.c_str (), value.c_str ());
```

Comment:
There's an issue here with parentheses, but that's also a call to a C function -- leaving `NULL` here may be appropriate, or maybe not. I don't think I care one way or another.